dot/core/service_integration_test.go

```diff
@@ -445,6 +444,7 @@ func TestHandleChainReorg_WithReorg_Transactions(t *testing.T) {
 	require.Equal(t, transaction.NewValidTransaction(tx, validity), pending[0])
 }
 
+// TODO: add test against latest gssmr runtime
```

Comment:
Should we create an issue for these several new `// TODO`s across the integ test files, so we can link it up?

Reply:
Yes I will create issues for them